### PR TITLE
ISPN-5735 Remove references to PermGen setting in JVM configurations …

### PR DESCRIPTION
**Pull request by Sanne**
…of tests, enlarge some heap sizes

https://issues.jboss.org/browse/ISPN-5735

### CODE REVIEW

File: documentation/src/main/asciidoc/contributing/chapter-5-Running_and_Writing_Tests.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@ Tests are written using the link:$$http://testng.org/$$[TestNG] framework.
 === Running the tests
 Before running the actual tests it is highly recommended to configure adjust suite's memory setting by updating the `MAVEN_OPTS` variables. E.g.
 
- $ export MAVEN_OPTS="-Xms512m -Xmx2048m -XX:MaxPermSize=384m"
+ $ export MAVEN_OPTS="-Xms512m -Xmx2048m"
```
Comment (danberindei) [inline documentation/src/main/asciidoc/contributing/chapter-5-Running_and_Writing_Tests.adoc]: Should probably replace MAVEN_OPTS with MAVEN_FORK_OPTS

File: integrationtests/security-manager-it/pom.xml [hunk 2]
```diff
@@ -57,9 +57,7 @@
                      <value>${testNGListener}</value>
                   </property>
                </properties>
-               
-               <argLine>-Xmx1024m -XX:MaxPermSize=256m
-                  -Dsun.nio.ch.bugLevel</argLine>
```
Comment (danberindei) [inline integrationtests/security-manager-it/pom.xml]: Most of the integration tests run in the container anyway, so this should be irrelevant.

File: documentation/src/main/asciidoc/contributing/chapter-5-Running_and_Writing_Tests.adoc (hunk 1 above)
Comment (Sanne) [inline documentation/src/main/asciidoc/contributing/chapter-5-Running_and_Writing_Tests.adoc]: that's material for a different JIRA ;-)

**Comment by anistor** [comment]
Integrated in master. Thanks @Sanne!

**Pull request closed without merging**